boards/lunakey_pico/main.py

```diff
@@ -0,0 +1,121 @@
+import board
+
+import digitalio
+import pwmio
+import time
+
+from kb import KMKKeyboard
+from kmk.keys import KC
+from kmk.extensions.RGB import RGB
+from kmk.extensions.RGB import AnimationModes
+from kmk.modules.split import Split, SplitSide, SplitType
+from kmk.modules.layers import Layers
+from kmk.modules.modtap import ModTap
+
+led = digitalio.DigitalInOut(board.GP25)
+led.direction = digitalio.Direction.OUTPUT
+led.value = True
+
+keyboard = KMKKeyboard()
+keyboard.tap_time = 100
+
+layers_ext = Layers()
+modtap_ext = ModTap()
+
+# TODO Comment one of these on each side
+split_side = SplitSide.LEFT
+split_side = SplitSide.RIGHT
+
+data_pin = board.GP1 if split_side == SplitSide.LEFT else board.GP0
+data_pin2 = board.GP0 if split_side == SplitSide.LEFT else board.GP1
+
+split = Split(
+    split_side=split_side,
+    split_type=SplitType.UART,
+    split_flip=True,
+    data_pin=data_pin,
+    data_pin2=data_pin2
+)
+
+rgb_ext = RGB(
+    pixel_pin=board.GP6,
+    num_pixels=6,
+    animation_mode=AnimationModes.BREATHING_RAINBOW
+)
+
+keyboard.modules = [layers_ext, modtap_ext, split]
+keyboard.extensions.append(rgb_ext)
+
+if split_side == SplitSide.LEFT:
+    buzzer = pwmio.PWMOut(board.GP8, variable_frequency=True)
+    OFF = 0
+    ON = 2**15
+    buzzer.duty_cycle = ON
+    buzzer.frequency = 2000
+    time.sleep(0.2)
+    buzzer.frequency = 1000
+    time.sleep(0.2)
+    buzzer.duty_cycle = OFF
+
+_______ = KC.TRNS
+XXXXXXX = KC.NO
```

Comment:
See above comment. Same here.

Reply:
@kdb424 Actually, the KC.XXXXXXX has already been defined. But, the length of the KC.XXXXXXX is 10, so I'm feeling that this is too long because I wanted to become the length within 7. Therefore, I intend to change the XXXXXXX to KC.NO.

Reply:
I removed this line at 0d27901.

Reply:
See above comment.